### ISSUE TITLE
[BUG] Neither README nor website have any example to show me what dynamoose can do for me

### ISSUE THREAD

**Issue by ankon** [dynamoose/dynamoose#1550]
### Summary:

I got a hint to look at this project from 24 Pull Requests, and so the first thing I did was open the README and try to figure out a simple question: "What is this? Why do I want it?"

README doesn't answer that, it just lists "features". The README does point to the website, so I started reading that ("Getting started" sounds like a good start!). After the first 5 pages or so I simply gave up. Getting started told me about "TypeScript support (in beta)" and "JSDoc support" ... but not a single example, snippet, or anything.

The first time I saw a bit of code was when looking at the "Guide" for the "Schema" type (randomly picked that), but those examples don't show the context or how to use dynamoose, they show how to configure a schema. No answer as to "Why do I need that?" or "How does that fit into the big picture?"

I certainly may have missed something, but I'm pretty sure it's not obvious :/

### Code sample:

N/A (documentation bug)

### Current output and behavior (including stack trace):

N/A (documentation bug)

### Expected output and behavior:

N/A (documentation bug)

### Environment:

N/A (documentation bug)

### Other information (if applicable):

This is a documentation problem.

### Other:
- [x] I have read through the Dynamoose documentation before posting this issue
- [x] I have searched through the GitHub issues (including closed issues) and pull requests to ensure this issue has not already been raised before
- [ ] I have searched the internet and Stack Overflow to ensure this issue hasn't been raised or answered before
- [ ] I have tested the code provided and am confident it doesn't work as intended
- [ ] I have filled out all fields above
- [ ] I am running the latest version of Dynamoose

... N/A?

**Comment by bestickley** [comment]
I agree. Having an "end to end" example would be really helpful in the docs.

**Comment by fishcharlie** [comment]
Great feedback. Dynamoose v1 had a much better example on the front page: https://v1.dynamoosejs.com.

Mongoose doesn't have the most complete example on their homepage, but still better than ours: https://mongoosejs.com
